pkgs/development/python-modules/brainflow/default.nix

```
@@ -0,0 +1,65 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  brainflow,
+  cmake,
+  nptyping,
+  numpy,
+  setuptools,
+  typish,
+  pythonOlder,
+}:
+buildPythonPackage {
+  pname = "${brainflow.name}-python";
+  inherit (brainflow) version;
+
+  disabled = pythonOlder "3.7";
+
+  src = "${brainflow.src}/python_package";
+
+  propagatedBuildInputs = [
+    numpy
+    (nptyping.overrideAttrs (finalAttrs: let
+      version = "1.4.4";
+    in {
+      inherit version;
+      src = fetchFromGitHub {
+        owner = "ramonhagenaars";
+        repo = "nptyping";
+        rev = "refs/tags/v${version}";
+        hash = "sha256-c9Qoufn9m3H03Pc8XhGzTBeixnl/elkalv50OrW4gJY=";
+      };
+
+      propagatedBuildInputs = finalAttrs.propagatedBuildInputs ++ [ typish ];
+
+      disabledTestPaths = [
+        "tests/test_functions/test_get_type.py"
+        "tests/test_functions/test_py_type.py"
+      ];
+    }))
+    setuptools
+  ];
+
+  buildInputs = [ brainflow ];
+
+  postInstall = ''
+    mkdir -p "$out/lib/$(ls $out/lib)/site-packages/${brainflow.pname}/lib/"
```

```suggestion
    mkdir -p "$out/${python.sitePackages}/brainflow/lib/"
```
Please don't use `pname` here.
It will break if pname is changed by override or similar.